cilium-cli/clustermesh/clustermesh.go

```diff
@@ -1653,62 +1700,122 @@ func (k *K8sClusterMesh) ConnectWithHelm(ctx context.Context) error {
 	}
 
 	// Enable clustermesh using a Helm Upgrade command against our target cluster
-	k.Log("ℹ️ Configuring Cilium in cluster '%s' to connect to cluster '%s'",
-		localClient.ClusterName(), remoteClient.ClusterName())
+	k.Log("ℹ️ Configuring Cilium in cluster %s to connect to cluster %s",
+		localClient.ClusterName(), strings.Join(remoteClusterNames, ","))
 	_, err = helm.Upgrade(ctx, localClient.HelmActionConfig, upgradeParams)
 	if err != nil {
 		return err
 	}
+	if k.params.ConnectionMode == "bidirectional" {
```

Comment:
Aren't those mutually exclusive? if so it might be best to switch based on mode. 
Also if bd, unicast, etc.. are interesting it would be great to use constants